services/task-svc/internal/task/task.go

```diff
@@ -579,15 +578,6 @@ func (ServiceServer) AssignTaskToUser(ctx context.Context, req *pb.AssignTaskToU
 		Str("userID", userId.String()).
 		Msg("user assigned")
 
-	if currentUserId, err := common.GetUserID(ctx); err == nil {
```

Comment:
also why?

Reply:
The initial usage of `hwes` in `task-svc` was just testing, not for event sourcing purposes.